File: deep_link_test/app/src/main/java/com/lazypotato/deeplinktest/MainActivity.java
package com.lazypotato.deeplinktest;

import androidx.appcompat.app.AppCompatActivity;

import android.net.Uri;
import android.os.Bundle;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Uri uri = getIntent().getData();
        if(uri != null) {
            String path = uri.toString();

            Toast.makeText(MainActivity.this, "Path: " + path, Toast.LENGTH_SHORT).show();
        }
    }
}

File: mvvm-hilt-flow-room/app/src/main/java/com/lazypotato/mvvm_hilt_flow_room/ui/MainActivity.kt
package com.lazypotato.mvvm_hilt_flow_room.ui

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.lazypotato.mvvm_hilt_flow_room.R
import dagger.hilt.android.AndroidEntryPoint

@AndroidEntryPoint
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}